src/main/scala/com/tapad/docker/ComposeTestRunner.scala

```diff
@@ -42,8 +42,9 @@ trait ComposeTestRunner extends SettingsHelper with PrintFormatting {
    * @param state The sbt state
    * @param args The command line arguments
    * @param instance The running Docker Compose instance to test against
+   * @param extraEnv extra environment variables to be passed to scalatest (and ConfigMap)
    */
-  def runTestPass(implicit state: State, args: Seq[String], instance: Option[RunningInstanceInfo]): State = {
+  def runTestPass(implicit state: State, args: Seq[String], instance: Option[RunningInstanceInfo], extraEnv: Map[String, String]): State = {
```

Comment:
One the update is made to call the new Task code directly from within `runTestPass` this extra parameter can be removed.